app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    NotWearingMaskError.masks_to_buy = 0
```

Comment:
it would be better to make `masks_to_buy ` as a normal variable, it will make your code simpler and more obvious to undestand